boards/BMS/Src/App/states/App_InitState.c

```diff
@@ -14,12 +13,13 @@ static void InitStateRunOnEntry(struct StateMachine *const state_machine)
     struct BmsCanTxInterface *can_tx_interface = App_BmsWorld_GetCanTx(world);
     struct Clock *            clock            = App_BmsWorld_GetClock(world);
     struct Accumulator *      accumulator      = App_BmsWorld_GetAccumulator(world);
-    struct OkStatus *         bms_ok_status    = App_BmsWorld_GetBmsOkStatus(world);
 
     App_CanTx_SetPeriodicSignal_STATE(can_tx_interface, CANMSGS_BMS_STATE_MACHINE_STATE_INIT_CHOICE);
     App_SharedClock_SetPreviousTimeInMilliseconds(clock, App_SharedClock_GetCurrentTimeInMilliseconds(clock));
     App_Accumulator_InitRunOnEntry(accumulator);
-    App_OkStatus_Enable(bms_ok_status);
+
+    // Set the BMS Ok status high
+    HAL_GPIO_WritePin(BMS_OK_GPIO_Port, BMS_OK_Pin, GPIO_PIN_SET);
```

Comment:
Not sure if it is the best idea to include GPIO Port and Pin information within the state machine code. This applies to the remaining PRs as well